html/elements/a.json

```
@@ -161,10 +161,12 @@
           "__compat": {
             "support": {
               "chrome": {
-                "version_added": "14"
+                "version_added": "14",
+                "notes": "Starting in Chrome 65, cross-origin downloads are not supported on the anchor element."
```

Suggesting `<a>` instead of _anchor_ for consistency with other notes in this file

```suggestion
                "notes": "Starting in Chrome 65, cross-origin downloads are not supported on the <code>&lt;a&gt;</code> element."
```